pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -687,6 +688,26 @@ public RebalanceResult rebalance(
     }
   }
 
+  @GET
+  @Path("/tables/{tableName}/rebalance/status")
+  @ApiOperation(value = "Table rebalance status", notes = "Get segments without the same state in EV and IS ")
+  public String getTableRebalanceStatus(
+      @ApiParam(value = "Name of the table", required = true) @PathParam("tableName") String tableName,
+      @ApiParam(value = "OFFLINE|REALTIME", required = true) @QueryParam("type") String tableTypeStr) {
+    try {
+        String tableNameWithType = constructTableNameWithType(tableName, tableTypeStr);
```

Comment:
(code format) Please reformat the changes with [Pinot Style](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#setup-ide)